documentation.html

```diff
@@ -0,0 +1,1440 @@
+            <label class="inputgroup">
+              <input class="input-textbox focus-blue" placeholder=" " />
+              <span class="input-label">Name</span>
+            </label>
+            <label class="inputgroup error">
+              <input class="input-textbox" placeholder=" " />
+              <span class="input-label">Name</span>
+              <span class="error-text">Error in your input</span>
+            </label>
+          </div>
+          <pre class="m-1-0"><code class="html border-round">
+&lt;label class="inputgroup"&gt;
+  &lt;input class="input-textbox focus-blue" placeholder=" " /&gt;
+  &lt;span class="input-label"&gt;Name&lt;/span&gt;
+&lt;/label&gt;
+
+&lt;label class="inputgroup error"&gt;
+  &lt;input class="input-textbox" placeholder=" " /&gt;
+  &lt;span class="input-label"&gt;Name&lt;/span&gt;
+  &lt;span class="error-text"&gt;Error in your input&lt;/span&gt;
+&lt;/label&gt;
+
+</code></pre>
+        </div>
+        <div id="image">
+          <h1 class="mt-2 uppercase letter-spaced">Images</h1>
+          <div class="grid-container web-two mob-one mt-1">
+            <div>
+              <img
+                src="https://images.unsplash.com/photo-1588392382834-a891154bca4d?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1955&q=80"
+                alt=""
+                class="responsive-image"
+              />
+            </div>
+            <div class="center">
+              <img
+                src="https://images.unsplash.com/photo-1588392382834-a891154bca4d?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1955&q=80"
+                alt=""
+                class="width-200px height-200px circle"
+              />
+            </div>
+          </div>
+          <pre class="m-1-0"><code class="html border-round">
+&lt;div&gt;
+  &lt;img src="" alt="" class="responsive-image" /&gt;
+&lt;/div&gt;
+
+&lt;div class="center"&gt;
+  &lt;img src="" alt="" class="width-200px height-200px circle" /&gt;
+&lt;/div&gt;
+
+</code></pre>
+        </div>
+        <div id="modals">
+          <h1 class="mt-2 uppercase letter-spaced">Modal</h1>
+          <button class="btn btn-col btn-secondary mt-1" id="modal-show">
+            See Modal
+          </button>
+          <p class="font-size-sm medium mt-05">Paste this code in your body</p>
+          <pre class="m-1-0"><code class="html border-round">
+&lt;div class="modal-overlay" id="modal"&gt;
+  &lt;div class="modal-container p-1 border-round"&gt;
+    &lt;h1 class="center font-size-l bold"&gt;Are you sure, you are a robot?&lt;/h1&gt;
+    &lt;div class="modal-buttons"&gt;
+      &lt;button class="btn btn-col btn-danger border-round" id="modal-close"&gt;
+        No !!
+      &lt;/button&gt;
+      &lt;button class="btn btn-col btn-success border-round" id="modal-success"&gt;
+        Yes
+      &lt;/button&gt;
+    &lt;/div&gt;
+  &lt;/div&gt;
+&lt;/div&gt;
+
```

Comment:
javascript code missing for modal functionality

Reply:
The cdn for JS is provided